app/categorization/llm_categorization.py

```diff
@@ -18,24 +18,29 @@ def Diagnosis(
         {"column": key, "content": value}
     )
     reply = str(llm_response_Diagnosis)
-    print(reply)
-
+    # print(reply)
+    
     if "yes" in reply.lower():
-        values = value.split()
-        unique_values = list(set(values[1:]))
-
-        # Create dictionary for Levels
-        levels_dict = {val: "" for val in unique_values}
+        output = {"TermURL": "nb:Diagnosis", "Levels": {}}
+        unique_entries=list_terms(key,value)
+        levels={} #the empty dictionary passed to the diagnosis_level function to be filled 
+        level={} # the dictionary which will become the output 
+        level = Diagnosis_Level(unique_entries, code_system,levels)
+        print(''' 
 
-        # Create the output dictionary
-        output = {"TermURL": "nb:Diagnosis", "Levels": levels_dict}
+llm_file
 
+''')
```

Comment:
Could we remove the print statement here?

Reply:
yes will do that